src/@utils/feedback.ts

```diff
@@ -1,89 +1,13 @@
-export const feedback: { [key in number]: string } = {
-  99: 'Decrypting file URL...',
-  0: '1/3 Looking for data token. Buying if none found...',
-  1: '2/3 Transfering data token.',
-  2: '3/3 Payment confirmed. Requesting access...'
-}
-
-export const publishFeedback: { [key in number]: string } = {
-  0: '1/5 Creating datatoken ...',
-  2: '2/5 Encrypting files ...',
-  4: '3/5 Storing ddo ...',
-  6: '4/5 Minting tokens ...',
-  8: '5/5 Asset published succesfully'
-}
-
-// TODO: do something with this object,
-// consumeStep should probably return one of those strings
-// instead of just a number
-export const consumeFeedback: { [key in number]: string } = {
-  ...feedback,
-  3: '3/3 Access granted. Consuming file...'
+// TODO: can be better
```

Comment:
think quick win would be to make it more generic as a method accepting `datatokenSymbol` & `baseTokenSymbol` and then switch them in the text